[2022-06-20 20:41:26,914] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl-processing.transform manual__2022-06-20T20:41:10.894220+00:00 [queued]>
[2022-06-20 20:41:27,027] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl-processing.transform manual__2022-06-20T20:41:10.894220+00:00 [queued]>
[2022-06-20 20:41:27,028] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 20:41:27,029] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-20 20:41:27,029] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 20:41:27,217] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform> on 2022-06-20 20:41:10.894220+00:00
[2022-06-20 20:41:27,277] {standard_task_runner.py:52} INFO - Started process 3978 to run task
[2022-06-20 20:41:27,311] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl-processing', 'transform', 'manual__2022-06-20T20:41:10.894220+00:00', '--job-id', '695', '--raw', '--subdir', 'DAGS_FOLDER/etl_processing.py', '--cfg-path', '/tmp/tmpb4eqtwzt', '--error-file', '/tmp/tmp7fhkcc2z']
[2022-06-20 20:41:27,313] {standard_task_runner.py:80} INFO - Job 695: Subtask transform
[2022-06-20 20:41:28,221] {task_command.py:370} INFO - Running <TaskInstance: etl-processing.transform manual__2022-06-20T20:41:10.894220+00:00 [running]> on host fe209149cfc1
[2022-06-20 20:41:29,295] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl-processing
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2022-06-20T20:41:10.894220+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-20T20:41:10.894220+00:00
[2022-06-20 20:41:29,700] {python.py:173} INFO - Done. Returned value was: None
[2022-06-20 20:41:29,914] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl-processing, task_id=transform, execution_date=20220620T204110, start_date=20220620T204126, end_date=20220620T204129
[2022-06-20 20:41:30,141] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-20 20:41:30,642] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
